src/analyzer.py

```diff
@@ -138,3 +140,38 @@ def noun_phrase(input_text):
     for chunk in doc.noun_chunks:
         n_phrase_lst.append(str(chunk))
     return n_phrase_lst
+
+
+def senti(tokens_column, sign):
```

Comment:
I would change the naming of the function. `senti` is too general. Perhaps `top_polarized_word`, so something that would reflect the purpose of this function.

Additionally, although I like the use of `sign` here, `1` and `-1` are kind of too generic and not intuitive. It could just be `pos` or `neg`. That way it would also make the if logic below more self-explaining than `1` or `-1`.